bases/rsptx/interactives/runestone/activecode/css/codemirror-dark.less

```
@@ -5,7 +5,7 @@
 
 // Force some colors to match those used by PreTeXt for syntax highlighting
 .cm-s-default .cm-comment { color: #3a812e; }
-.cm-s-default .cm-keyword { color: rgb(18, 137, 201); }
+.cm-s-default .cm-keyword { color: #067CBC; }
```

```suggestion
.cm-s-default .cm-keyword { color: #0679B7; }
```
Third time's the charm.